### DISCUSSION TITLE
How install and play on Mac OS for non IT specialist?

### DISCUSSION THREAD

**Discussion by oleshkoigor** [Try/OpenGothic#431] [Q&A]
Please write how to install everything? :) I'm not an IT specialist and I don't understand what to do. I feel like this guide is missing.

**I copy paste this in terminal:**

brew install glslang
git clone --recurse-submodules https://github.com/Try/OpenGothic.git
cd OpenGothic
cmake -H. -Bbuild -DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo
cmake --build ./build --target Gothic2Notr

**#** locate the executables at OpenGothic/build/opengothic

**Terminal installed something and then wrote:**

zsh: command not found: cmake
zsh: command not found: cmake
zsh: command not found: #

Something went wrong apparently. And I can't figure out where to copy the game itself from GOG? Do I need to unpack the game or copy the GOG installation file to some specific folder? Where do I need to copy the original game?

app from Mac Source Ports do not work :)

Maybe somewhere there is a step-by-step instruction and I just do not see it? If there is, can you provide a link? Please help me figure it out?  🙏

**Comment by Try** [comment]
Hi, @oleshkoigor !

We slowly working towards making project more user friendly, but not quite there yet :)

AFAIR, cmake on mac can be installed by simply typing `brew install cmake`

> Where do I need to copy the original game?

In principle anywhere, you just need to specify this path later, when launching. In my case command line is:
`./Gothic2Notr.sh -g /Users/alexanderveselov/steam/gothic2`

> app from Mac Source Ports do not work 

They assume that GOG instalation path is at:`~/Library/Application Support/OpenGothic`

**Reply by oleshkoigor** [reply]
Super! It works! Game launch! But the mouse does not move properly :(

**Reply by Romubay** [reply]
i think the issue with the mouse is described in the discussions here. Otherwise try this build with the launcher: https://macsourceports.com/game/gothic2 
I personally use it as well, until Try and the Team finishes their own launcher :)

**Reply by Try** [reply]
Mouse should be fixed already: https://github.com/Try/OpenGothic/issues/417

**Comment by damahaaa** [comment]
I copied all this and everything seems to be installed successfully

brew install glslang
git clone --recurse-submodules https://github.com/Try/OpenGothic.git
cd OpenGothic
cmake -H. -Bbuild -DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo
cmake --build ./build --target Gothic2Notr

what should I do next? How to install the game? how to run it then?

**Reply by Try** [reply]
> what should I do next?

Hi, you need to locate executable file in build folder and then run it:
`./Gothic2Notr.sh -g /path/to/gothic2/files`

**Reply by damahaaa** [reply]
i tried like this `./Gothic2Notr.sh -g /Users/zhenzhen/Downloads/Gothic2GoldGOG/RUS/setup_gothic_2_gold_russian_2.7_(14553).exe` 

and writes to me 
zsh: no matches found: /Users/zhenzhen/Downloads/Gothic2GoldGOG/RUS/setup_gothic_2_gold_russian_2.7_(14553).exe

**Reply by damahaaa** [reply]
I'm sorry, I don't understand code (

**Reply by damahaaa** [reply]
in general, this is the path to the installation file /Users/zhenzhen/Downloads/Gothic2GoldGOG/RUS/setup_gothic_2_gold_russian_2.7_(14553).exe

**Reply by damahaaa** [reply]
if i write like this `./Gothic2Notr.sh -g /Users/zhenzhen/Downloads/Gothic2GoldGOG/RUS/setup_gothic_2_gold_russian_2.7_(14553)`

and writes to me
zsh: unknown file attribute: 1

**Reply by Try** [reply]
> /Users/zhenzhen/Downloads/Gothic2GoldGOG/RUS/setup_gothic_2_gold_russian_2.7_(14553).exe

This is not an installation path.

From the Readme:
```
Common Gothic installation paths to be provided via -g:
    "C:\Program Files (x86)\JoWooD\Gothic II"
    "C:\Gothic II"
    "C:\Program Files (x86)\Steam\steamapps\common\Gothic II"
    "~/PlayOnLinux's virtual drives/Gothic2_gog/drive_c/Gothic II"
```

GoG manual page tell that games are installed to ` ~/Library/Application Support/<gamename>` by default

**Reply by damahaaa** [reply]
I didn't understand anything at all( 
could you write step by step what i need to do next?

**Reply by Try** [reply]
:(

Can you tell to what folder did you install gothic2 game files ?

**Reply by damahaaa** [reply]
> :(
> 
> Can you tell to what folder did you install gothic2 game files ?

I can't install them because MacOS can't install .exe

**Reply by damahaaa** [reply]
> in general, this is the path to the installation file /Users/zhenzhen/Downloads/Gothic2GoldGOG/RUS/setup_gothic_2_gold_russian_2.7_(14553).exe

and the path to the installation file is

**Comment by damahaaa** [comment]
<img width="510" alt="Снимок экрана 2023-05-09 в 23 20 05" src="https://github.com/Try/OpenGothic/assets/132818280/e1c3cb3e-c633-4a21-9e0f-75ddd9f86cc7">

all I have

**Reply by Try** [reply]
than you don't have game file at all seemingly. Gothic folder should appear like this:
![изображение](https://github.com/Try/OpenGothic/assets/122894/796c0718-29e4-4019-971d-fbbdf7d936ed)

`_work` and `Data`, are one of most important parts here

**Reply by damahaaa** [reply]
this is a folder with the game already installed, but I can't install it on macOS ..

**Reply by 0xF16** [reply]
> this is a folder with the game already installed, but I can't install it on macOS ..

I have done it today for the first time. Maybe it requires some additional app being installed, but try to install PortingKit and install Gothic with it (it's the app to emulate Windows for games on MacOS). When you have Gothic installed (I have a GOG version), go to Finder -> Applications -> Gothic.app (right-click on it) -> Show Package Contents:
<img width="322" alt="image" src="https://github.com/Try/OpenGothic/assets/94185094/a865eb19-c260-4090-bcd4-bc05fcc46e82">
Then, `drive_c/GOG Games/Gothic` directory.
Now you have 2 options:
- You can either copy all it's content to OpenGothic `build/openghotic`
- from the `build/openghotic` run the compiled OpenGhotic executable like: `./Gothic2Notr.sh -g ~/Applications/Gothic.app/drive_c/GOG\ Games/Gothic` (so you do not have to have to copy files)

Edit1: Oh, and I have just realized you might be looking for Gothic II. I was doing that from Gothic I, but the approach should be pretty similar. I want to finish first game before trying to finish Gothic II after like ~17 years.
